Mathlib/CategoryTheory/Abelian/Transfer.lean

```diff
@@ -20,6 +22,10 @@ then `C` is also abelian.
 
 See <https://stacks.math.columbia.edu/tag/03A3>
 
+## enough-injectives
+If `C, D` are abelian categories with adjoint functors `L ⊣ R` where `L` is a faithful exact
```

Comment:
It's not clear in this docstring whether `L` or `R` is a functor from `C` to `D`.

Reply:
Add more explanation